cloudca/resource_cloudca_instance.go

```diff
@@ -266,9 +275,8 @@ func resourceCloudcaInstanceDelete(d *schema.ResourceData, meta interface{}) err
 	if _, err := ccaResources.Instances.Destroy(d.Id(), true); err != nil {
 		if ccaError, ok := err.(api.CcaErrorResponse); ok {
 			if ccaError.StatusCode == 404 {
-				fmt.Errorf("Instance %s does no longer exist", d.Get("name").(string))
 				d.SetId("")
-				return nil
+				return fmt.Errorf("Instance %s does no longer exist", d.Get("name").(string))
```

Comment:
If you are trying to delete, and it is not found, I think you should log an ignore, not error out. If the instance no longer exists, why make TF fail?